### DISCUSSION TITLE
[02/01/2025] Score Community Meeting

### DISCUSSION THREAD

**Discussion by sujaya-sys** [score-spec/spec#116] [Community meetings]
First community call of 2025! 🎉

**Info**

| Community call | Info |
|:-----------|:------------|
| Meeting Link | Join via [Google Meet](https://meet.google.com/znt-usdc-hzs) or call +49 40 8081618260 (Pin: 599 887 196)
| Meeting Agenda & Notes | Add to our agenda or review minutes [here](https://github.com/score-spec/spec/discussions/categories/community-meetings)
| Meeting Time | 1:00-2:00pm UTC, every first Thursday of the month

**Open To Dos**

- [ ] @sujaya-sys to write up and discuss with Florian how to quantify or drive Score adoption (GHA action usage, CLIs usage, Website and Docs traffic)
--> Set up knowledge base, reach out to users e.g. via surveys, check with the community on how they deal with this
- [x] @sujaya-sys to align with Ben and Andrej on next steps for Azure implementation 
- [ ] @sujaya-sys @chrishumanitec Implement decision making process on new issues as discussed [here](https://github.com/score-spec/spec/discussions/107#discussioncomment-11178592)
---> Check in again in 1-2 weeks
---> Look at common best practices (e.g. pre-releases for implementation), check how other projects deal with releases of specs
- [ ] @chrishumanitec  Create issue to turn files into key value pair rather than array
---> Still outstanding

**Rollovers from the last call**

- [x] Let's file issue(s) to support a new `score-* resources available-provisioners` command generating in the output this table or a more advanced/detailed version with `-o json`: https://docs.score.dev/docs/score-implementation/score-compose/resources-provisioners/? In `score-go` with that it's common across all the implementations?
- [ ] Let's add a new "Examples" entry in the docs where we port all of the examples from there https://github.com/score-spec/score-compose/tree/main/examples?
  - 1 page per Score file?
  - Where to store the Score files (because common to `score-compose` and `score-k8s`)? In a new dedicated `examples` repo?
  - Where do we show the `score-compose init|generate`+`compose up` or `score-k8s init|generate`+`kubectl apply -f`? Under this section https://docs.score.dev/docs/score-implementation/? Or on each example page with 2 tabs (`score-compose` and `score-k8s`)?
  - Thoughts: Have more generic (not specific to score-compose) examples could be a good start.
---> https://github.com/score-spec/spec/tree/main/samples (validate against the schema -> reference from the docs)

**Comment by sujaya-sys** [comment]
Started a write up here on how to track Score adoption: https://docs.google.com/document/d/1qxYQofJ2yDOfm2ZoayYjyzvues_Dn1fSduWG49iYYHs/edit?tab=t.0 cc @FL303 @mathieu-benoit - Feedback is welcome

**Comment by sujaya-sys** [comment]
Call for feedback on https://github.com/score-spec/spec/issues/104 Issue: https://cloud-native.slack.com/archives/C07DN0D1UCW/p1734640299195289

**Comment by mathieu-benoit** [comment]
Hi @sujaya-sys and team, JFYI I won't be able to attend on Jan 2nd.

I think most of us are in holidays, so I'm wondering if we should postpone this one to the week after? Or maybe cancelling this one and resuming 2025 with the one already planned in Feb?

**Reply by astromechza** [reply]
@sujaya-sys / @chrishumanitec sorry I missed it too!

**Comment by mathieu-benoit** [comment]
As per our discussions, now tracking these two:
- https://github.com/score-spec/score-k8s/issues/112
- https://github.com/score-spec/score-compose/issues/237
